src/Agent/NewRelic/Agent/Core/Segments/DatastoreSegmentData.cs

```diff
@@ -239,5 +258,17 @@ public void SetConnectionInfo(ConnectionInfo connInfo)
         {
             _connectionInfo = connInfo;
         }
+
+        // FOR UNIT TESTING ONLY
```

Comment:
Should we use the `///` comment style here so that the intellisense will include this message too?

Reply:
Sure.